src/fabric/src/fabric2_fdb.erl

```diff
@@ -314,12 +317,65 @@ delete(#{} = Db) ->
     } = ensure_current(Db),
 
     DbKey = erlfdb_tuple:pack({?ALL_DBS, DbName}, LayerPrefix),
-    erlfdb:clear(Tx, DbKey),
-    erlfdb:clear_range_startswith(Tx, DbPrefix),
+    DoRecovery = fabric_util:do_recovery(),
+    case DoRecovery of
+        true ->
+            {Mega, Secs, _} = os:timestamp(),
+            NowSecs = Mega * 1000000 + Secs,
```

Comment:
This could be `erlang:system_time(second).` I think?

Reply:
yes, thanks @nickva for comments.

I changed this locally with 
```
            {{Y, Mon, D}, {H, Min, S}} = calendar:universal_time(),
            NowSecs = list_to_binary(lists:flatten(
                io_lib:format("~w~2.10.0B~2.10.0B."
                ++ "~2.10.0B~2.10.0B~2.10.0B",
                    [Y, Mon, D, H, Min, S])
            )),
            DeletedDbKey = erlfdb_tuple:pack({?DELETED_DBS, DbName, NowSecs},
                LayerPrefix),
```
and will push new commit today with other changes.

Reply:
Makes sense. There is a balance here between using a non-standard format for time and the timestamp being an identifier of an db instance we want to restore. That is on one hand it might be nice to use https://en.wikipedia.org/wiki/ISO_8601 format (`2007-03-01T13:00:00Z` for ex.), however since we'd be passing that in the URL path we'd have to deal with `:` being a reserved character and encode it and all that.

Reply:
It looks to be fine to use ISO_8601 format, i.e. using `:` in URL.
```
curl -u foo:bar  http://127.0.0.1:15984/db01 -X GET
{"update_seq":"000000000000000000000000","cluster":{"n":0,"q":0,"r":0,"w":0},"compact_running":false,"data_size":0,"db_name":"db01","disk_format_version":0,"disk_size":0,"instance_start_time":"0","purge_seq":0,"doc_del_count":0,"doc_count":0,"sizes":{"external":2,"views":0}}
curl -u foo:bar  http://127.0.0.1:15984/db01 -X DELETE
{"ok":true}
curl -u foo:bar  http://127.0.0.1:15984/db01/_deleted_dbs_info
"[\"2020-03-20T11:42:54Z\"]"
curl -u foo:bar  http://127.0.0.1:15984/db01?recover=2020-03-20T11:42:54Z -X PUT
{"ok":true}
```

Reply:
hey @nickva, with the suggestion from @mikerhodes, the timestamp is specified in payload of POST instead of URL
```
  POST /{db}/_restore

  { "sourceTS": "<deletedTS>", "destination": "animaldb"}
```
Thus, it is probably safe to use ISO_8601 format, like `2007-03-01T13:00:00Z` where `:` exists in payload.